### PR TITLE
Add bloomberg to member org and dsun to members

### PR DESCRIPTION
**Pull request by yuzisun**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/community/245)

### CODE REVIEW

File: member_organizations.yaml [hunk 1]
```diff
@@ -46,6 +46,14 @@
     - kimwnasptd@arrikto.com
   url: https://www.arrikto.com
 
+- name: Bloomberg
+  contacts:
+    - dsun20@bloomberg.net
+    - klaban1@bloomeberg.net
+    - ifilonenko@bloomberg.net
+  url: http://www.bloomberg.com
+  hiringUrl: https://www.bloomberg.com/careers
```
Comment (abhi-g) [inline member_organizations.yaml]: Do we really need the hiringUrl in here?

**Comment by abhi-g** [comment]
@yuzisun I'd prefer if we not pollute that file with hiring solicitation :)

File: member_organizations.yaml (hunk 1 above)
Comment (yuzisun) [reply member_organizations.yaml]: no, let me take it off.

Comment (abhi-g) [reply member_organizations.yaml]: thanks

**Pull request merged**